docs/secpoll.zone

```diff
@@ -517,4 +517,5 @@ dnsdist-1.8.0-rc2.security-status                          60 IN TXT "2 Unsuppor
 dnsdist-1.8.0-rc3.security-status                          60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
 dnsdist-1.8.0.security-status                              60 IN TXT "1 OK"
 dnsdist-1.8.1.security-status                              60 IN TXT "1 OK"
```

Comment:
1.8.0 and 1.8.1 are no longer OK

Reply:
That's not really true, is it? They are OK when linked against a patched h2o library.

Reply:
So technically our packages are not OK, but I don't think we can make this distinction in secpoll, unfortunately.

Reply:
That is true, but the 1.8.1 package from repo.powerdns.com will never be linked against a patched library, because it's already built and released. secpoll is not used much outside of our packages, as far as I know

Reply:
Done